src/main/java/core/basesyntax/FruitsFileReader.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FruitsFileReader {
+
+    public List<FruitListOperation> read(String filename) {
+        List<FruitListOperation> operations = new ArrayList<>();
+        try (BufferedReader reader
+                     = new BufferedReader(
+                             new FileReader("src/main/java/core/basesyntax/" + filename))) {
+            reader.readLine(); // пропускаєм перший рядок
```

Comment:
lets remove comments from code, if you really need them, you should write then in english